server/opts.go

```diff
@@ -178,7 +178,9 @@ type Options struct {
 	Gateway               GatewayOpts   `json:"gateway,omitempty"`
 	LeafNode              LeafNodeOpts  `json:"leaf,omitempty"`
 	JetStream             bool          `json:"jetstream"`
-	StoreDir              string        `json:"store_dir"`
+	JetStreamMaxMemory    int64         `json:"-"`
+	JetStreamMaxStore     int64         `json:"-"`
+	StoreDir              string        `json:"-"`
```

Comment:
Maybe rename this to JetStreamStoreDir?

Reply:
I think store is ok, since we may add in stuff to support K/V.